docs/troubleshooting/harvester.md

```diff
@@ -4,7 +4,37 @@ sidebar_label: Harvester
 title: ""
 ---
 
-## Generate a support bundle
+## Fail to Deploy Multi-node Cluster due to Incorrect HTTP Proxy Setting
+
+### Configure HTTP Proxy During Harvester Installation
+
+In some environments, you configure [HTTP Proxy](../airgap/#configure-an-http-proxy-during-installation) during Harvester installation.
+
+### One Node Becomes Unavailable
+
+The issue you may encounter:
+
+```
+The first node is installed successfully.
+
+The second node is installed successfully.
+
+The third node is installed successfully.
+
+Then the second node changes to Unavialable state and cannot recover automatically.
+```
+
+### Solution
+
+When the nodes in the cluster are not using the HTTP Proxy to communicate with each other, after the first node is installed successfully, you need to login into the `Harvester GUI` to configure [no-proxy](../airgap.md#configure-an-http-proxy-in-harvester-settings) against the CIDR used by those nodes.
```

Comment:
it can also be configured via https://docs.harvesterhci.io/v1.1/install/harvester-configuration#system_settings during the installation.